mingw-w64-wxSVG/PKGBUILD

```
@@ -1,16 +1,18 @@
 # Maintainer: Jordan Irwin <antumdeluge@gmail.com>
 
 _realname=wxsvg
-_wx_basever=3.0
-pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+_wx_basever=3.2
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}-wx${_wx_basever}"
 pkgbase="mingw-w64-${_realname}"
 pkgver=1.5.23
-pkgrel=2
+pkgrel=1
 pkgdesc="A C++ library to create, manipulate and render SVG files (mingw-w64)"
 arch=(any)
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
 url="https://wxsvg.sourceforge.io/"
 license=("custom:wxWindows")
+provides=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+conflicts=("${MINGW_PACKAGE_PREFIX}-${_realname}")
```

```suggestion
conflicts=("${MINGW_PACKAGE_PREFIX}-${_realname}")
replaces=("${MINGW_PACKAGE_PREFIX}-${_realname}")
```

I think we should move users over, no?

---

Oh, the [documentation I read](https://wiki.archlinux.org/title/PKGBUILD#replaces) said to use `provides` & `conflicts` if just uploading an alternative version of a package. But I guess I misunderstand the use of `replaces`.

---

Done.

---

I means all existing users of wxsvg will be automatically moved over to this package, and the old one will be uninstalled. It's somewhat questionable since they may not be compatible, but we don't have any users in MSYS2 itself, so hard to say.

The alternative is that user will stay on the old package and when they try to install wxsvg-wx3.2 it will complain about the conflict and let them switch.